Makefile

```diff
@@ -2,7 +2,7 @@
 TEST_INFRA_VERSION ?= latest
 
 # Version of gRPC core used for the gRPC driver
-DRIVER_VERSION ?= v1.58.1
+DRIVER_VERSION ?= v1.60.0
```

Comment:
Cound you change this version to `v1.62.1`? Thanks!

Reply:
Done, thank you!